dlgs_linux.go

```diff
@@ -45,23 +42,19 @@ func (b *MsgBuilder) error() {
 }
 
 func (b *FileBuilder) load() (string, error) {
-	return chooseFile("Load", gtk.FILE_CHOOSER_ACTION_OPEN, b)
+	return chooseFile("Open File", "Open", gtk.FILE_CHOOSER_ACTION_OPEN, b)
 }
 
 func (b *FileBuilder) save() (string, error) {
-	f, err := chooseFile("Save", gtk.FILE_CHOOSER_ACTION_SAVE, b)
+	f, err := chooseFile("Save File", "Save",  gtk.FILE_CHOOSER_ACTION_SAVE, b)
 	if err != nil {
 		return "", err
 	}
-	_, err = os.Stat(f)
-	if !os.IsNotExist(err) && !Message("%s already exists, overwrite?", filepath.Base(f)).yesNo() {
-		return "", Cancelled
-	}
 	return f, nil
 }
 
-func chooseFile(title string, action gtk.FileChooserAction, b *FileBuilder) (string, error) {
-	dlg, err := gtk.FileChooserDialogNewWith2Buttons(firstOf(b.Dlg.Title, title), nil, action, "Ok", gtk.RESPONSE_ACCEPT, "Cancel", gtk.RESPONSE_CANCEL)
+func chooseFile(title string, buttonText string, action gtk.FileChooserAction, b *FileBuilder) (string, error) {
+	dlg, err := gtk.FileChooserDialogNewWith2Buttons(firstOf(b.Dlg.Title, title), nil, action,  "Cancel", gtk.RESPONSE_CANCEL, buttonText, gtk.RESPONSE_ACCEPT)
```

Comment:
Hm, does this affect which action is selected if the user just presses enter? (ie. the dialog's default focus)

Reply:
Just tested it, and it does not change the behavior. Both cases just does the ok-action.